ref/Texts_and_Tables.md

```diff
@@ -150,18 +150,15 @@ It returns a boolean value.
 
 #### Drawing tables: `drawtable(‹vec›,‹list›)`
 
-**Not available in CindyJS yet!**
-
 One- and two-dimensional lists can be easily drawn in a geometric view as tables.
 In the simplest form one has only to provide the list and a position where the table should be drawn.
 Modifiers can be used to fine tune the appearance of the table.
 
 **Example:**
 The following code produces the picture below:
 
-    - skip test: drawtable not implemented yet
-    > x=1..10;
-    > table=apply(x,(#,#^2,#^3,#^4));
+    > x=1..4;
+    > table=apply(x,(#,#^2,#^3));
```

Comment:
I guess that I had thought about adding draw command tests here, and wanted to keep the number of commands low, but then decided against it again. By the way, this just made me think of #460, an idea which I've been toying with for quite some time now.